core/src/main/java/org/radargun/stages/cache/test/AbstractQueryStage.java

```diff
@@ -72,7 +103,8 @@ public String toString() {
 
         @Override
         public void apply(Queryable.QueryBuilder builder) {
-            builder.eq(path, value);
+            super.apply(builder);
```

Comment:
Declaring another protected abstract void apply(QueryBuilder, SelecExpression) would be probably more elegant than requiring to call super.apply(builder), but that's rather a matter of taste.